Scripts/RemediationScripts/Remediate-ConfigureConditionalAccessPolicyForPIM.ps1

```diff
@@ -1,43 +1,47 @@
 ﻿<###
 # Overview:
-    This script is used to configure Conditional Access (CA) policy to require PIM elevation from SAW for admin roles (Owner, Contributor, UAA) in a Subscription.
+    This script is used to configure Conditional Access (CA) policy to require PIM elevation from SAW for all built-in roles and all custom roles except for a few built-in roles in a Subscription.
```

Comment:
policy to require --> policy to restrict

Reply:
Fixed as suggested.